ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vxlan_interface.schema.yml

```
@@ -20,6 +20,8 @@ keys:
               source_interface:
                 type: str
                 description: Source Interface Name
+              multicast_headend_replication:
+                type: bool
```

Given that there are other keys under mutlicast in EOS and we don't have implemented any yet, 
```
ceos1(config-if-Vx1)#show cli commands | grep "vxlan multicast"
[no|default] vxlan multicast ( ipv4 | ipv6 ) [ disabled ]
[no|default] vxlan multicast headend-replication
[no|default] vxlan multicast protocol pim ROUTE_TYPE
[no|default] vxlan multicast routing ipv4
[no|default] vxlan multicast-group decap { DECAP }
```

let's change the schema to the following for future growth (you do not need to implement the other keys in this PR)

```suggestion
              multicast:
                type: dict
                keys:
                  headend_replication:
                    type: bool
```

---

updated the schema